vector/v.voronoi/skeleton.c

```diff
@@ -525,7 +525,7 @@ int tie_up(void)
 	                         IPoints[i]);
 	}
 
-	distmin = 1. / 0.; /* +inf */
+	sscanf("inf", "%lf", &distmin);
```

Comment:
In #2681, the code was changed to use `INFINITY`, but there's a note that it might not for MSVC (at that time)

```c
        distmin = INFINITY;
```